doc/source/contributing.rst

```diff
@@ -84,3 +84,93 @@ This way, it's not possible for you to push code that fails the style checks::
   trim trailing whitespace.................................................Passed
   Add License Headers......................................................Passed
   Validate GitHub Workflows................................................Passed
+
+Build the documentation
+-----------------------
+
+.. note::
+
+  In order to build the documentation, you must have the Geometry Service
+  installed and running on your machine. The documentation is built using
+  the Geometry Service to generate examples. It is also recommended that the
```

Comment:
```suggestion
  installed and running on your machine because it is used to generate the
  examples in the documentation. It is also recommended that the
```